curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f805f813eaf2049bc2ceea.md

```diff
@@ -26,25 +26,29 @@ assert.equal(document.querySelectorAll('label input')?.length, 4);
 You should add the first `input` after the `label` text `Enter Your First Name:`, and include a space after the colon.
 
 ```js
-assert.equal(document.querySelectorAll('label')?.[0]?.innerHTML.trim(), 'Enter Your First Name: <input>');
+const query = /^Enter Your First Name:\s+<input>/
+assert.equal(query.test(document.querySelectorAll('label')?.[0]?.innerHTML.trim()),true);
 ```
 
 You should add the second `input` after the `label` text `Enter Your Last Name:`, and include a space after the colon.
 
 ```js
-assert.equal(document.querySelectorAll('label')?.[1]?.innerHTML.trim(), 'Enter Your Last Name: <input>');
+const query = /^Enter Your Last Name:\s+<input>/
+assert.equal(query.test(document.querySelectorAll('label')?.[1]?.innerHTML.trim()),true);
 ```
 
 You should add the third `input` after the `label` text `Enter Your Email:`, and include a space after the colon.
 
 ```js
-assert.equal(document.querySelectorAll('label')?.[2]?.innerHTML.trim(), 'Enter Your Email: <input>');
+const query = /^Enter Your Email:\s+<input>/
+assert.equal(query.test(document.querySelectorAll('label')?.[2]?.innerHTML.trim()),true);
 ```
 
 You should add the fourth `input` after the `label` text `Create a New Password:`, and include a space after the colon.
 
 ```js
-assert.equal(document.querySelectorAll('label')?.[3]?.innerHTML.trim(), 'Create a New Password: <input>');
+const query = /^Create a New Password:\s+<input>/
+assert.equal(query.test(document.querySelectorAll('label')?.[3]?.innerHTML.trim()),true);
```

Comment:
```suggestion
assert.match(document.querySelectorAll('label')?.[3]?.innerHTML.trim(), query);
```

Reply:
Done.

Reply:
This one appears to still be assert.equal

Reply:
Thanks Naomi. I missed that one by accident.